persistence-utils-demo-test/pom.xml

```diff
@@ -48,13 +48,6 @@
 				<version>10.0.0</version>
 				<extensions>true</extensions>
 			</plugin>
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-surefire-plugin</artifactId>
-				<configuration>
-					<reuseForks>false</reuseForks>
```

Comment:
switching back to re-useable forks is indeed a big performance boost -50 seconds test execution time :star_struck: 
![re-usedjvm](https://user-images.githubusercontent.com/15085693/199947190-994fd689-8991-4738-9b2e-a84d564aeb07.png)